src/components/Account/AccountModule.js

```diff
@@ -79,10 +83,10 @@ function AccountModule({ compact }) {
 
   const { direction, screenIndex } = useMemo(() => {
     const screenId = (() => {
+      if (account) return 'connected'
```

Comment:
Changing the other not will avoid show error screen when it happen?

Reply:
This was already there I didn't change 🤷🏼‍♂️

Reply:
It show to me the order changed, the `if (error)` was in first place.

![image](https://user-images.githubusercontent.com/691510/155819270-cf0e224c-c799-4cee-830b-9ad4a53530b9.png)